.travis.yml

```diff
@@ -11,7 +10,7 @@ env: MINIMUM_VERSIONS=false
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
+    - php: 7.0
```

Comment:
Ah, I didn't mean to switch to PHP 7 as part of this PR... but I guess that's okay. Change is going to require a major version bump anyways.